finder-app/writer.c

```diff
@@ -0,0 +1,44 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <syslog.h>
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <fcntl.h>
+
+int main(int argc, char *argv[])
+{
+	FILE *fd; //file descriptor
+	
+	openlog(NULL, 0, LOG_USER);
+	
+	//check for correct number of input arguments
+	if(argc != 3){
+		syslog(LOG_ERR, "Wrong number of arguments provided. Exiting...\n");
+		perror("Wrong number of arguments provided. Exiting...\n");
```

Comment:
@prvbharadwaj1731 think about how you can help the user do the right thing here...